scss/civicrm/campaign/_campaign.scss

```diff
@@ -0,0 +1,43 @@
+.page-civicrm-campaign {
+  .crm-content-block {
+    @include box-shadow($box-shadow);
+
+    .dataTables_wrapper,
+    .crm-form-block {
+      box-shadow: none;
+    }
+
+    .dataTables_wrapper {
+      margin-bottom: 0;
+
+      > table {
+        thead th:nth-child(2),
+        tbody td:nth-child(2) {
+          padding-left: 20px;
+        }
+      }
+    }
+
+    > #mainTabContainer {
+      border-radius: $border-radius-base;
+
+      .ui-tabs-nav {
+        border-radius: $border-radius-base $border-radius-base 0 0;
+      }
+
+      .ui-tabs-panel {
```

Comment:
why just for this page?

Reply:
outer container should have 20px padding + elements like accordion or table should stick to this panel. so, we are adding this in specific to this page.

Reply:
can't we make it generic then, for any page with the structure of this specific page?

Reply:
no this is much riskier given that we do not have any parent class identifying to have page structure is like this.

In general cases, tab panels do have content or forms, in our case we have tables and accordions.
Also, all elements are having generic classes so, it makes it more problematic to handle by creating general scenarios.